src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
```

Comment:
I would love to see propTypes for this component (it would get to use the `DateTime` type too)